frontend/apps/pwa/src/features/recorder/index.ts

```diff
@@ -0,0 +1 @@
+export { RecordEditorContainer } from './RecordEditor.container';
```

Comment:
```
export { RecordEditorContainer as RecordEditor  } from './RecordEditor.container';
```
Components using RecordEditor don't care that it's a container. Exporting like this allows us to refactor it to not use a container in the future if we want.